styles/sass/partials/_base.scss

```
@@ -1,8 +1,23 @@
 body {
+  position: relative;
   min-height: 100vh;
   display: grid;
   grid-template-rows: auto 1fr auto;
   grid-template-columns: 100%;
+  // Body background applied to a before element so that a 'filter' can be applied to the background, but not the rest of the body.
```

Great use of before pseudo element!
May I suggest linear-gradient on the background-image though?
```suggestion
background-image: linear-gradient(90deg, rgba(40, 40, 40) 80%, rgba(35, 35, 35) 80%), url();
```